data/json/recipes/chem/fuel.json

```diff
@@ -0,0 +1,128 @@
+[
+  {
+    "result": "chem_ethanol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "brewing_cookbook", 4 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 2, "amount": 1 },
+        { "id": "BREW", "level": 2, "amount": 1 }
+     ],
+     "tools": [
+       [ [ "fire", -1 ], [ "hotplate", 30 ], [ "toolset", 30 ] ],
+       [ [ "thermometer", -1 ] ]
```

Comment:
I meant the one that spawns in cities and broadcasts weather on radio. It's more of a radio station, but it sounds rather counterintuitive to me that a station that broadcasts weather wouldn't have a local thermometer (as a backup at the very least).